x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useState, useEffect, Fragment } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiDataGrid,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { ErrorCallout } from '../error_callout';
+import {
+  getDependentVar,
+  getPredictionFieldName,
+  loadEvalData,
+  loadDocsCount,
+  DataFrameAnalyticsConfig,
+} from '../../../../common';
+import { getTaskStateBadge } from '../../../analytics_management/components/analytics_list/columns';
+import { DATA_FRAME_TASK_STATE } from '../../../analytics_management/components/analytics_list/common';
+import {
+  isResultsSearchBoolQuery,
+  isClassificationEvaluateResponse,
+  ClassificationEvaluateResponse,
+  ResultsSearchQuery,
+  ANALYSIS_CONFIG_TYPE,
+} from '../../../../common/analytics';
+import { LoadingPanel } from '../loading_panel';
+import { getColumnData } from './column_data';
+
+interface Props {
+  jobConfig: DataFrameAnalyticsConfig;
+  jobStatus: DATA_FRAME_TASK_STATE;
+  searchQuery: ResultsSearchQuery;
+}
+
+export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery }) => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [confusionMatrixData, setConfusionMatrixData] = useState<
+    ClassificationEvaluateResponse['classification']['multiclass_confusion_matrix']['confusion_matrix']
```

Comment:
This could be `ConfusionMatrix[]` if you adapt the suggestion further up to split the types into multiple interfaces.